manifests/cf-manifest/spec/cloud-config/env_spec.rb

```diff
@@ -0,0 +1,16 @@
+
+RSpec.describe "Environment specific configuration" do
+  let(:default_cloud_config) { cloud_config_with_defaults }
+  let(:prod_cloud_config) { render_cloud_config("prod") }
+
+  def get_disk_size(cloud_config, disk_name)
+    cloud_config["disk_types"].select { |j| j["name"] == disk_name }.first["disk_size"]
+  end
+
+  it "should specify a larger elasticseach disk size in production" do
```

Comment:
Syntax error in regex. I think you meant `s/elasticseach/elasticsearch/` :trollface: